benchmarks/benchmark_qt_viewer.py

```diff
@@ -0,0 +1,22 @@
+# See "Writing benchmarks" in the asv docs for more information.
+# https://asv.readthedocs.io/en/latest/writing_benchmarks.html
+# or the napari documentation on benchmarking
+# https://github.com/napari/napari/blob/master/BENCHMARKS.md
+import napari
+from qtpy.QtCore import QTimer
+from qtpy.QtWidgets import QApplication
+
+
+class QtViewerSuite:
+    """Benchmarks for viewing images in the viewer."""
+
+    def setup(self):
+        app = QApplication.instance() or QApplication([])
+        self.viewer = None
+
+    def teardown(self):
+        self.viewer.window.close()
+
+    def time_create_viewer(self):
+        """Time to create the viewer."""
+        self.viewer = napari.Viewer()
```

Comment:
This is fantastic. Even if we don't understand by itself where the time is going, it's enough to know whether this value is going up or not. Super valuable.

Reply:
yeah, i'll also add this number is around 200ms, but typing napari on the cli and then waiting leaves about 7 seconds before an i see the empty viewer, so lots still to understand

Reply:
Perhaps the QApplication takes a while? That should be timed also I guess!